com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/util/ApplicationURITest.java

```diff
@@ -1,14 +1,25 @@
 package com.sap.cloud.lm.sl.cf.core.util;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.stream.Stream;
 
+import org.cloudfoundry.client.lib.domain.CloudDomain;
+import org.cloudfoundry.client.lib.domain.CloudRoute;
+import org.cloudfoundry.client.lib.domain.ImmutableCloudDomain;
+import org.cloudfoundry.client.lib.domain.ImmutableCloudRoute;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
+import com.sap.cloud.lm.sl.cf.core.model.SupportedParameters;
+
 public class ApplicationURITest {
 
+    private static final String CUSTOM = "custom-";
```

Comment:
Rename to "CUSTOM_PREFIX"

Reply:
On second thought, "CUSTOM" fits better with the variables and it is easier to read this way.
You decide if you want to rename it

Reply:
It seems better for me to be just CUSTOM